Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM elixir:1.7.3-slim
+
+RUN adduser --disabled-password --gecos '' test
+USER test
```

Comment:
If we are only testing and not deploying the image there is no need to make it more secure by using a different user.